platform/core/src/feature_categories.rs

```diff
@@ -236,12 +236,48 @@ mod tests {
     }
 
     impl FileSystem for MockFileSystem {
-        fn read_contents_of_file(&self, path: &str) -> Result<Vec<u8>, CoreFailure> {
+        fn file_content(&self, path: &str) -> Result<Vec<u8>, CoreFailure> {
```

Comment:
Maybe call this TestFileSystem or suchlike, so that it's clear even if you see a fragment of the file (which is the case)